twisted/test/test_text.py

```diff
@@ -181,66 +181,66 @@ def test_multiWord(self):
 
 class StrFileTests(unittest.TestCase):
     def setUp(self):
-        self.io = StringIO("this is a test string")
+        self.stream = StringIO(u"this is a test string")
```

Comment:
No, I'm not going to use BytesIO, because that will require the use of byte strings.

What I can do though is to prefix all strings in this test with u"", so that they
will be unicode strings on both Python 2 and Python 3.